app/scripts/app.js

```diff
@@ -192,11 +192,15 @@ angular.module('confRegistrationWebApp', ['ngRoute', 'ngCookies', 'ui.bootstrap'
                  * server side so the users access_token can be fetched to build the log out URL.
                  * after the GET, if successful, then delete the cookies. */
               } else if ($cookies.crsAuthProviderType === 'FACEBOOK') {
-                $http.get('auth/facebook/logout').success(function (data, status, headers) {
+                $http.get('auth/facebook/logout').success(function () {
```

Comment:
Okay, I added calls to the API for relay and no auth tokens.  I need to build those methods on the API though (tomorrow) so this won't work on stage yet.. but you could still review the code? :)